resources/assets/sass/components/_news.scss

```diff
@@ -0,0 +1,110 @@
+// News
+//
+// News blocks
+//
+// Markup:
+//<div class="c-news {$modifiers}">
+//    <a href="#" class="c-news__image-wrap">
+//        <img src="assets/img/news1.jpg" alt="" class="c-news__image">
+//    </a>
+//    <h3 class="c-news__title">
+//        <a href="#" class="c-news__title-link">Euroopa Kohus otsustas – lennuki tehniline rike ei päästa hüvitise maksmisest</a>
+//    </h3>
+//    <div class="c-news__meta">
+//        <p class="c-news__meta-date">Täna 12:31</p>
+//    </div>
+//</div>
+//
+// default   - A news block.
+// .m-small   - A small news block.
+//
+// Styleguide 6
+
+
+.c-news {
+
+	margin-bottom: 24px;
+
+	@include respond-to(tablet) {
+
+		margin-bottom: 48px;
+	}
+}
+
+	.c-news__image-wrap {
+
+		position: relative;
+		overflow: hidden;
+		display: block;
+		height: 192px;
+		margin-bottom: 24px;
+
+		.c-news.m-small & {
+
+			height: 144px;
+		}
+	}
+
+		.c-news__image {
+
+			position: absolute;
```

Comment:
Properties should be ordered display, height, left, max-width, position, top, transform